nautobot/docs/models/extras/dynamicgroup.md

```diff
@@ -1,7 +1,319 @@
+- **Exclude (`NOT`)** - The **Exclude** operator performs a set _difference_ on the queryset, and is equivalent to a Boolean `NOT`. Any objects matching the child filter are _excluded_ (aka _differenced_) from the preceding filter. Any matching objects from the child filter will be negated from the members of the resultant filter.
+
+The following table maps the Nautobot **operator** to the corresponding set operations:
+
+| Operator | Set Operation | Boolean | Description                                                  |
+| -------- | ------------- | ------- | ------------------------------------------------------------ |
+| Restrict | Intersection  | AND     | All criteria from parent/child filters _must_ match          |
+| Include  | Union         | OR      | Any criteria from parent/child filters _may_ match           |
+| Exclude  | Difference    | NOT     | All matches from child filter are _negated_ from parent filter |
+
+Any filters provided by the child groups are used to filter the members from the parent group using one of the three operators: **Restrict (AND)**, **Include (OR)**, or **Exclude (NOT)**. This allows for logical parenthetical grouping of nested groups by the operator you choose for that child group association to the parent.
+
+Using the example group hierarchy from above, let's apply operators and explain how it would work:
+
+```no-highlight
+parent {filter: None}
+- first-child {weight: 10, operator: intersection, filter: site=ams01}
+- second-child {weight: 20, operator: union, fitler: site=ang01}
+- third-child {weight: 30, operator: difference, filter: None}
+  - nested-child {weight: 10, operator: intersection, filter: status=active}
+```
+
+Logically, the filter will be expressed like so using the hierarchy above:
+
+```no-highlight
+((first-child OR second-child) AND (NOT nested-child))
+```
+
+#### How does this work?
+
+First, the filters for direct children for a group will always be included in a parenthetical grouping, separated by the operator. Any _empty_ filters are skipped as they imply a parent group that contains other groups. Therefore the filter of `parent` and the filter of `third-child` are not directly used for generating the filter. The filter from `parent` is passed through from left to right as the "base" filter.
+
+Similarly, by the time we get to `third-child`, the filter at that node in the graph will have already had the filter from `first-child` and `second-child` applied, and will merely be passed through to the filter generated from the children of `third-child`.
+
+Because `first-child` and `second-child` are "included" by way of the `union` set operator (Boolean `OR`), but because the filter for `third-child` is empty as explained above, its filter condition is not directly included, resulting in `(first-child OR second-child)` as the first group, representing all _direct_ filters for the group `parent`.
```

Comment:
Confusing here as `OR` is specified for `second-child` only, but `first-child` uses `AND`, yet here the first sentence seems to imply that both children are using `OR`. See my suggestion below about explicitly including "all objects" as the starting point for the parent filter as perhaps a clearer approach.

Also I don't know that distinguishing between "direct" and "indirect" filters adds any clarity to the discussion?